src/cljs/rems/table.cljs

```diff
@@ -161,27 +161,41 @@
 (defn- filterable? [column]
   (:filterable? column true))
 
-(defn- display-row? [row filtered-columns filters]
-  (if (empty? filtered-columns)
-    true ; table has no filtering enabled
-    (some (fn [column]
-            (str/includes? (str (get-in row [(:key column) :filter-value]))
-                           filters))
-          filtered-columns)))
+(defn- display-row? [row filtered-columns search-terms]
+  (or (empty? filtered-columns) ; table has no filtering enabled
+      (empty? search-terms)
+      (every? (fn [search-term]
+                (some (fn [column]
+                        (str/includes? (str (get-in row [(:key column) :filter-value]))
+                                       search-term))
+                      filtered-columns))
```

Comment:
you could factor out the filtered-columns -> values-of-filtered-columns computation. it would improve readability (and I guess performance too)